faqs/galaxy/interactive_tools_jupyter_launch.md

```diff
@@ -9,13 +9,15 @@ layout: faq
 Depending on which server you are using, you may be able to run JupyterLab directly in Galaxy. If that is not available, [Try JupyterLab](https://jupyter.org/try) can be an alternative.
 
 > ### {% icon tip %} Tip: Launch JupyterLab in Galaxy
-> Currently JupyterLab in Galaxy is available on [Live.useGalaxy.eu](https://live.usegalaxy.eu)
+> Currently JupyterLab in Galaxy is available on [Live.useGalaxy.eu](https://live.usegalaxy.eu), [usegalaxy.org](https://usegalaxy.org) and [usegalaxy.eu](https://usegalaxy.eu).
 >
-> 1. Open the JupyterLab tool {% icon tool %} by clicking [here](https://live.usegalaxy.eu/?tool_id=interactive_tool_jupyter_notebook){:target="_blank"}
-> 2. Click Execute
-> 3. The tool will start running and will stay running permanently
-> 4. Click on the "User" menu at the top and go to "Active Interactive Tools" and locate the JupyterLab instance you started.
-> 5. Click on your JupyterLab instance
+> > ### {% icon hands_on %} Hands-on: Run JupyterLab
+> >
+> > 1. {% tool [Interactive Jupyter Notebook](interactive_tool_jupyter_notebook) %}:
+> > 2. Click Execute
+> > 3. The tool will start running and will stay running permanently
+> > 4. Click on the "User" menu at the top and go to "Active Interactive Tools" and locate the JupyterLab instance you started.
+> > 5. Click on your JupyterLab instance
```

Comment:
```suggestion
> > 5. Click on your JupyterLab instance
> {: .hands_on}
```

Reply:
Good shout! Thank you! I fixed it manually because I was looking at a jekyll instance and it needed a another line too and reformatting.